articles/how-to-consume-data-from-an-API-using-react/index.md

```diff
@@ -0,0 +1,424 @@
+
+In this tutorial, we will be using [OMDb API](http://www.omdbapi.com/). You need an API key to gain access to this API. This API key can be generated from the OMDb API's [website](http://www.omdbapi.com/apikey.aspx). 
+
+![signup](/engineering-education/how-to-consume-data-from-an-API-using-react/signup.png)
+
+The API key will be sent to your email, as shown below.
+
+![email](/engineering-education/how-to-consume-data-from-an-API-using-react/email.png)
+
+> Note that the API key must be included in the API request, as shown:
+
+```
+http://www.omdbapi.com/?i=tt3896198&apikey={your_key}
+```
+
+When we navigate to the above link in a browser, the endpoint will return the JSON data as shown below.
+
+```json
+{
+   "Title":"Guardians of the Galaxy Vol. 2",
+   "Year":"2017",
+   "Rated":"PG-13",
+   "Released":"05 May 2017",
+   "Runtime":"136 min",
+   "Genre":"Action, Adventure, Comedy, Sci-Fi",
+   "Director":"James Gunn",
+   "Writer":"James Gunn, Dan Abnett (based on the Marvel comics by), Andy Lanning (based on the Marvel comics by), Steve Englehart (Star-Lord created by), Steve Gan (Star-Lord created by), Jim Starlin (Gamora and Drax created by), Stan Lee (Groot created by), Larry Lieber (Groot created by), Jack Kirby (Groot created by), Bill Mantlo (Rocket Raccoon created by), Keith Giffen (Rocket Raccoon created by), Steve Gerber (Howard the Duck created by), Val Mayerik (Howard the Duck created by)",
+   "Actors":"Chris Pratt, Zoe Saldana, Dave Bautista, Vin Diesel",
+   "Plot":"The Guardians struggle to keep together as a team while dealing with their personal family issues, notably Star-Lord's encounter with his father the ambitious celestial being Ego.",
+   "Language":"English",
+   "Country":"USA",
+   "Awards":"Nominated for 1 Oscar. Another 15 wins & 57 nominations.",
+   "Poster":"https://m.media-amazon.com/images/M/MV5BNjM0NTc0NzItM2FlYS00YzEwLWE0YmUtNTA2ZWIzODc2OTgxXkEyXkFqcGdeQXVyNTgwNzIyNzg@._V1_SX300.jpg",
+   "Ratings":[
+      {
+         "Source":"Internet Movie Database",
+         "Value":"7.6/10"
+      },
+      {
+         "Source":"Rotten Tomatoes",
+         "Value":"85%"
+      },
+      {
+         "Source":"Metacritic",
+         "Value":"67/100"
+      }
+   ],
+   "Metascore":"67",
+   "imdbRating":"7.6",
+   "imdbVotes":"562,796",
+   "imdbID":"tt3896198",
+   "Type":"movie",
+   "DVD":"N/A",
+   "BoxOffice":"$389,813,101",
+   "Production":"Marvel Studios, Walt Disney Pictures",
+   "Website":"N/A",
+   "Response":"True"
+}
+```
+
+In the above `JSON` response, we have data like the `Title,` `Year,` `Rated,` `Released,` `Runtime,` `Genre`, etc, of the movie Guardians of the Galaxy Vol. 2.
+
+### Step 2: Creating a react project.
+
+Let's create a React application using the following command:
+
+```
+npx create-react-app OMDbApi
+```
```

Comment:
Newer versions of `create-react-app` don't support app names with an uppercase letter. Provide an app name with lowercase letters like `omdbapi`.

Also, syntax highlighting is missing. mark it as `bash`.